com.regnosys.rosetta.tests/src/com/regnosys/rosetta/tests/RosettaExpressionsTest.xtend

```diff
@@ -48,9 +48,11 @@ class RosettaExpressionsTest {
 		model => [
 			((elements.last as Function)
 			  .operations.head.expression as RosettaConditionalExpression) => [
-			  	isFull.assertFalse;
-			  	elsethen.assertNotNull;
-			  	eqHelper.equals(elsethen, ExpressionFactory.eINSTANCE.createListLiteral).assertTrue
+			  	assertFalse(isFull);
+			  	assertNotNull(elsethen);
+			  	val lit = ExpressionFactory.eINSTANCE.createListLiteral
+			  	lit.generated = true
```

Comment:
I added a property `generated` to `RosettaExpression` to see whether the expression was generated by the `RosettaDerivedStateComputer` as syntactic sugar or not.